README.md

```diff
@@ -51,16 +51,36 @@ about this tool as well as e2e testing generally.
 
 Create a PR in this repo to add/update/remove a job or suite. Specifically
 you'll need to do the following:
-* Create an entry in [`jobs/config.json`] for the job
-  - If this is a kubetest job create the corresponding `jobs/env/FOO.env` file
-  - It will pick a free project from [boskos](/boskos) pool by default, or
-  - You can also set --gcp-project=foo in [`jobs/config.json`] for a dedicated project, make sure the project has the right [IAM grants](jenkins/check_projects.py)
+* Add the job to the appropriate section in [`config/jobs`](config/jobs)
+  - Directory Structure:
+    - In general we follow config/jobs/org/repo/name-of-config-file.yaml
+    - For Kubernetes Repo we also allows config/jobs/kubernetes/sig-foo/name-of-config-file.yaml
+    - We use basename of the config name as a key in the prow configmap, so the name of your config file need to be unique across the config subdir.
+  - Type of jobs:
+    - Presubmit jobs run on unmerged code in PRs
+    - Postsubmit jobs run after merging code
+    - Periodic job run on a timed basis
+    - You can find more prowjob definitions at [how-to-add-new-jobs](prow#how-to-add-new-jobs)
+  - Scenario args:
+    - You can append scenario/kubetest args inline in your prowjob definition, example:
+    ```
+      - name: foo-repo-test
+        interval: 1h
+        agent: kubernetes
+        spec:
+          containers:
+          - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+            args:
+            - --repo=github.com/org/repo
+            - --timeout=90
+            - --scenario=execute
+            - --
+            - make
+            - test
+    ```
+
 * Add the job name to the `test_groups` list in [`testgrid/config.yaml`](testgrid/config.yaml)
```

Comment:
We also want a generator for this, I _always_ forget and it seems like a good job for a robot

Reply:
testgrid config can have more customized options, which probably cannot tell from a prowjob config?

Reply:
I mean a generator for this would be great, we can probably make it merge with custom options... but we can discuss that some other time.

Reply:
yeah I would defer on generator for now, IIRC testgrid is populated with more than just prow-driven jobs

Reply:
Yeah, not for this PR -- I just know that every time I add a new job to `test-infra` I need to re-read and re-grok the two (?) places in the test-grid config where I need to add it too. Good doc for this would be fine too, the test catches it, it just seems like the vast majority of the testgrid config looks identical, just reflecting the names of things. I don't think it makes sense to generate the entire config obviously but a tool that would update the config with the defaults (?) when you add a new presubmit etc would be nice

Reply:
We do support multiple testgrid configs as well. If we are fine, we can add some sort of tag on prowjob to annotate a testgrid config, otherwise we can generate a default one? cc @fejta